src/log.ts

```diff
@@ -1,13 +1,26 @@
 import { handlers, LevelName, Logger, LogRecord } from "std/log/mod.ts";
 import { format } from "std/datetime/mod.ts";
+import { join } from "std/path/mod.ts";
+import dirs from "dirs/mod.ts";
 import { isTesting } from "./_utils.ts";
+import { ErgomaticError } from "./error.ts";
 
 function formatter({ loggerName, levelName, msg }: LogRecord) {
   const datetime = format(new Date(), "yyyy-MM-dd HH:mm:ss");
 
   return `[${datetime}][${loggerName}][${levelName}] ${msg}`;
 }
 
+function _logsDir() {
```

Comment:
this is coming in separate PR (it ended up being a bit more involved), prefixed with `_` for now to silence linter.